contrib/inventory/gce.py

```diff
@@ -159,6 +159,18 @@ def get_config(self):
             config.add_section('inventory')
 
         config.read(gce_ini_path)
+
+        #########
+        # Section added for processing ini settings
+        #########
+
+        # filter on instance states
+        self.instance_states = []
+
+        if config.has_option('gce', 'instance_states'):
+            for inst_state in config.get('gce', 'instance_states').split(','):
```

Comment:
split already returns a list, so do we need do the append?  If you split a string without a comma, it would just return the (single) item in a list, so I think you can avoid the loop.